### ISSUE TITLE
Reset filter icon.

### ISSUE THREAD

**Issue by philipbaginski** [outl1ne/nova-detached-filters#30]
Hi!

First of all thanks for making this package alive. Great work! The only question I have, and it's not an issue, but I don't know how to make a pull request:

Is it not better - of course this is my point of view - to use old refresh icon instead of new one? The old one - refresh - looks better and gives more clear info what will happened. New one is visually unreadable.

The rest looks and works just outstanding :-)

**Comment by KasparRosin** [comment]
Hei!
Could you share a screenshot of what your reset icons look like?
I did not change any icons intentionally and on my test project, they look the same as they did in nova3.

![image](https://user-images.githubusercontent.com/33309407/176998203-c5ed54cc-e7eb-43fb-b9d6-8b444fb083dd.png)

**Comment by philipbaginski** [comment]
> Hi! Could you share a screenshot of what your reset icons look like? I did not change any icons intentionally and on my test project, they look the same as they did in nova3.
> 
> ![image](https://user-images.githubusercontent.com/33309407/176998203-c5ed54cc-e7eb-43fb-b9d6-8b444fb083dd.png)

I see the same icon like you. But old icon - regarding screenshots in the docs looks different, like standard refresh icon - see pictures in the docs.

**Comment by philipbaginski** [comment]
Here is the code of existing icon:

<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512.004 512.004" height="18" width="18" fill="currentColor">
    
    <path
      d="m361.183 0c-54.059 0-99.836 36.049-114.505 85.331h-192.948c-18.024 0-28.614 20.339-18.285 35.119.16.231-5.363-7.31 129.747 177.039 2.714 3.951 4.148 8.57 4.148 13.367v177.688c0 19.435 22.224 30.24 37.473 18.754l57.593-43.518c8.614-6.415 13.754-16.655 13.754-27.409v-125.515c0-4.798 1.435-9.417 4.149-13.369l46.497-63.451c76.139 21.439 151.81-36.022 151.81-114.791.001-65.752-53.577-119.245-119.433-119.245zm-103.192 279.919c-5.835 7.968-9.831 19.1-9.831 30.938 0 136.483.734 127.081-1.68 128.869-1.91 1.421 10.835-8.188-47.14 35.618v-164.488c0-11.012-3.327-21.608-9.622-30.646-.169-.242 4.923 6.71-120.835-164.88h172.938c-1.457 44.852 22.126 84.961 58.678 106.581zm103.192-71.428c-49.314 0-89.434-40.035-89.434-89.246 0-49.21 40.12-89.245 89.434-89.245 49.313 0 89.433 40.035 89.433 89.245.001 49.211-40.119 89.246-89.433 89.246z"
    />
    <path
      d="m400.201 80.298c-5.854-5.864-15.35-5.872-21.213-.02l-17.805 17.773-17.805-17.773c-5.863-5.853-15.361-5.846-21.213.02-5.853 5.862-5.844 15.36.019 21.213l17.767 17.735-17.767 17.735c-5.863 5.853-5.872 15.351-.019 21.213 5.833 5.844 15.331 5.891 21.213.02l17.805-17.773 17.805 17.773c5.845 5.834 15.343 5.862 21.213-.02 5.853-5.862 5.844-15.36-.019-21.213l-17.767-17.735 17.767-17.735c5.863-5.853 5.872-15.351.019-21.213z"
    />
  </svg>

, and here the old one (I took from Nova icon set, but it looks the same like old one):

<svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor" stroke-width="2">
  <path stroke-linecap="round" stroke-linejoin="round" d="M4 4v5h.582m15.356 2A8.001 8.001 0 004.582 9m0 0H9m11 11v-5h-.581m0 0a8.003 8.003 0 01-15.357-2m15.357 2H15" />
</svg>

**Comment by KasparRosin** [comment]
Ahh, there is slight mistake from my part here.
The one in docs is actually out-dated.
We changed the reset icon in release [1.0.4] ([Release Notes](https://github.com/outl1ne/nova-detached-filters/blob/1.0.4/CHANGELOG.md))
I'll update the docs with fresh nova 4 screenshots some time next week.

There was an internal discussion a while back, when this package was first developed, as to which icon to use and we opted to switch out the "refresh" icon, since it could have been misunderstood by new and existing nova users.

I don't think we'll change it back.
